sandbox/grommet-app/src/pages/layouts/kinds/HPEMarketing/MarketingBrochure.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { useContext } from 'react';
+import {
+  Box,
+  Button,
+  Grid,
+  Image,
+  Heading,
+  PageContent,
+  Text,
+  ResponsiveContext,
+} from 'grommet';
+import { LinkNext } from 'grommet-icons';
+import female from '/female.png';
+import clock from '/clock.svg';
+import performance from '/performance.svg';
+
+const ContentArea = ({ alt, image, desc, title }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  console.log(breakpoint);
+  return (
+    <Box align="center" gap="medium">
+      <Box
+        width={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+        height={breakpoint === 'medium' ? 'xxsmall' : 'xsmall'}
+      >
+        <Image fit="contain" alt={alt} src={image} />
```

Comment:
Why are we adding the svgs as images as opposed to just consuming the icons from grommet-icons?

Reply:
good question I can add icons from grommet-icons

Reply:
Icons were to small I left images for now

Reply:
Since the goal of this is to see where tokens are/aren't working, I'd push towards using icons and adding a comment that you couldn't achieve something near the desired size.

Reply:
gotcha I misunderstood let me switch them out!

Reply:
@taysea switched to icons and left a comment